### ISSUE TITLE
Add a button to stop any music/video playing from webpage

### ISSUE THREAD

**Issue by developer-diganta** [developer-diganta/Dino#149]
### Subject of the issue
A button can be added that stops any music/video playing from webpage.

**Comment by dheerajchhatanidc** [comment]
I want to work on this issue, also there is a typo on the label "iwoc".

**Comment by piyushmishraa** [comment]
Hello @developer-diganta  sir, I would love to work on this issue.can you please assign it to me?

**Comment by developer-diganta** [comment]
Hi @piyushmishraa ! Thank you for your interest. Just wanted to confirm if you are participating in IWOC?

**Comment by piyushmishraa** [comment]
@developer-diganta yes sir, I am a sophomore from kiet group of institution participating in IWOC 2023.
I have already contributed to one of the organization and would love to work on this issue.

**Comment by developer-diganta** [comment]
Sure! Go ahead!
